src/GeneralStateTestsFiller/stCreateTest/CREATE2_CallDataFiller.yml

```diff
@@ -0,0 +1,60 @@
+CREATE2_CallData:
+  _info:
+    comment: |
+      Test if calldata is empty in initcode context.
+  env:
+    currentCoinbase: a94f5374fce5edbc8e2a8697c15331677e6ebf0b
+    currentDifficulty: 1
+    currentGasLimit: 1000000
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      nonce: 0
+      balance: 100000000000000
+      storage: { }
+      code: ''
+
+    # Contract creator.
+    000000000000000000000000000000000c5ea705:
+      nonce: 0
+      balance: 0
+      storage: { }
+      code: |
+        :yul object "C" {
```

Comment:
this had a conflict with yul.sh args. now fixed. however your code works only with yul optimisations enabled.
`        :yul berlin optimise object "C" {`

Reply:
here https://github.com/chfast/tests/pull/1